app/models/package_manager/base.rb

```diff
@@ -227,6 +228,10 @@ def self.project_find_names(project_name)
       [project_name]
     end
 
+    def self.entire_package_deprecation_info(name)
```

Comment:
This is a bikeshed but I'm not loving this class method name. `entire_package_` feels unnecessary? We don't do most things at a version level and this method is on a project so by definition it's about the project.

So maybe this is just `self.deprecation_info`? Are we planning on using the `message` field?

Reply:
I'm fine with renaming it. We will use the `message` field in the next piece of deprecation information being passed on, which will come in a week or so.

Reply:
I'm fine with renaming it. We will be using the `message` field for work coming in a week or two.